### PR TITLE
Special pages test

### PR DESCRIPTION
**Pull request by jpita**

### CODE REVIEW

File: cypress/page-objects/article-preview-page.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export class ArticlePreviewPage {
+  getTitle () {
+    return cy.get('.article-preview .title')
+  }
+}
```
Comment (hueitan) [inline cypress/page-objects/article-preview-page.js]: you define this but didn't use it?

File: cypress/integration/special-pages-spec.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  beforeEach(() => {
+    cy.navigateToHomePage()
+  })
+
+  it('special page article should open', () => {
+    cy.visit('http://127.0.0.1:8080/#/article/en/Help%3AIPA%2FEnglish')
+    articlePage.title().should('have.text', 'Help:IPA/English')
+    cy.get('.article-content').should('contain.text', 'Throughout Wikipedia, the pronunciation of words is indicated by means of the ')
+  })
+})
```
Comment (hueitan) [inline cypress/integration/special-pages-spec.js]: not sure how special it is? are you going to test that opening the preview message, then goes to this article page?

File: cypress/page-objects/article-preview-page.js (hunk 1 above)
Comment (jpita) [reply cypress/page-objects/article-preview-page.js]: will use it when the pagination works on cypress, right now I can't select the link I want to open on the preview

File: cypress/integration/special-pages-spec.js (hunk 2 above)
Comment (jpita) [reply cypress/integration/special-pages-spec.js]: right now I can't select the link I want to open on the preview , will complement the test when I can

**Pull request merged**